### PR TITLE
Add links

### PR DESCRIPTION
**Pull request by ychoy**

### CODE REVIEW

**Review by cnwilliams — APPROVED** [review]

File: app/views/tips/_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<%= form_for @tip do |f| %>
+  <div class="form-group">
+ 		<div class="form-group">
```
Comment (nathanallen) [inline app/views/tips/_form.html.erb:3]: indentation

File: app/views/tips/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    	<%= f.label :title %>
```
Comment (nathanallen) [inline app/views/tips/_form.html.erb:4]: indentation

File: app/views/tips/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+      <%= f.text_field :title, placeholder: "Title", class: "form-control", autofocus: true %>
+    </div>
+    <div class="form-group">
+      <%= f.label :body %>
+      <%= f.text_field :body, placeholder: "Content", class: "form-control", autofocus: true %>
+    </div>
+    <div class="actions">
+			<%= f.submit "Save Post", class: "btn btn-block btn-primary" %>
```
Comment (nathanallen) [inline app/views/tips/_form.html.erb:12]: indentation

File: app/views/tips/show.html.erb [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
+     <h3><strong> <%= @author.first_name %> <%= @author.last_name %></strong></h3>
+     <p><strong>Date Joined: </strong><%= (@author.created_at).strftime("%m/%d/%Y") %></p>
+     <p><strong>City: </strong> <%= @author.current_city %></p>
+   </div>
+   <div class="col-md-8">
+     <h3><%= @tip.title %></h3>
+     <p><%= @tip.body %></p>
+     <br>
+     <button type="button" class="btn btn-info"><%= link_to "Edit", edit_tip_path(@tip) %></button>
+     <%= link_to "Delete", delete_tip_path(@tip), :method => 'get', :data => {:confirm => 'Are you sure you want to delete this post?'} %>
```
Comment (nathanallen) [inline app/views/tips/show.html.erb:16]: Remove `delete_tip` alias from routes. `delete_tip_path` describes the same path as `tip_path` (`tips#show`).

File: app/views/tips/show.html.erb [hunk 5]
```diff
@@ -1,10 +1,21 @@
...
+     </div>
+     <h3><strong> <%= @author.first_name %> <%= @author.last_name %></strong></h3>
+     <p><strong>Date Joined: </strong><%= (@author.created_at).strftime("%m/%d/%Y") %></p>
+     <p><strong>City: </strong> <%= @author.current_city %></p>
+   </div>
+   <div class="col-md-8">
+     <h3><%= @tip.title %></h3>
+     <p><%= @tip.body %></p>
+     <br>
+     <button type="button" class="btn btn-info"><%= link_to "Edit", edit_tip_path(@tip) %></button>
```
Comment (nathanallen) [inline app/views/tips/show.html.erb:15]: Use `button_to` helper.
see http://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to

File: app/views/users/_form.html.erb [hunk 6]
```diff
@@ -25,11 +25,9 @@
       <%= f.password_field :password, placeholder: "Password must be at least 6 characters", class: "form-control" %>
     </div>
     <div class="actions">
-			<%= f.submit "Sign Up", class: "btn btn-block btn-primary" %>
+			<%= f.submit "Save", class: "btn btn-block btn-primary" %>
```
Comment (nathanallen) [inline app/views/users/_form.html.erb:28]: indentation

File: app/views/users/show.html.erb [hunk 7]
```diff
@@ -1,43 +1,21 @@
...
-     <div class="col-md-8">
-       <h3><strong><%= @user.first_name.capitalize %>'s Profile</strong></h3>
-       <hr>
-       <p><strong>Name:</strong> <%= @user.first_name %> <%= @user.last_name %> </p>
-       <p><strong>Email:</strong> <%= @user.email %></p>
-       <p><strong>Current City:</strong> <%= @user.current_city %></p>
-         <p class="small text-muted">Joined on <%= (@user.created_at).strftime("%m/%d/%Y") %></p>
-     </div>
-   <% end %>
+     <h3><strong> <%= @user.first_name %> <%= @user.last_name %></strong></h3>
```
Comment (nathanallen) [inline app/views/users/show.html.erb:7]: mind your whitespace

File: app/views/users/show.html.erb [hunk 8]
```diff
@@ -1,43 +1,21 @@
...
+     <p><strong>Date Joined: </strong><%= (@user.created_at).strftime("%m/%d/%Y") %></p>
+     <p><strong>City: </strong> <%= @user.current_city %></p>
```
Comment (nathanallen) [inline app/views/users/show.html.erb:9]: mind your whitespace

File: app/views/users/show.html.erb [hunk 9]
```diff
@@ -1,43 +1,21 @@
...
+     <button type="button" class="btn btn-info"><%= link_to "Edit Profile", edit_user_path(@user) %></button>
```
Comment (nathanallen) [inline app/views/users/show.html.erb:10]: use `button_to`

**Comment by nathanallen** [comment]
make sure to add `.DS_Store` to your `.gitignore`

**Review by nathanallen — CHANGES_REQUESTED** [review]

**Review by mnfmnfm — COMMENTED** [review]
I second Nathan

**Pull request closed without merging**